pkg/services/auth/auth_token.go

```diff
@@ -404,6 +414,36 @@ func (s *UserAuthTokenService) rotatedAfterParam() int64 {
 	return getTime().Add(-s.Cfg.LoginMaxInactiveLifetime).Unix()
 }
 
+func (s *UserAuthTokenService) revokeOldUserTokens(ctx context.Context, userId int64, maxConcurrentSessions int) error {
+	return s.SQLStore.WithDbSession(ctx, func(dbSession *sqlstore.DBSession) error {
+		sql := `
+		DELETE
+		FROM user_auth_token
+		WHERE id NOT IN
+			(
+			  SELECT id
+			  FROM user_auth_token
+			  WHERE user_id = ?
+			  ORDER BY seen_at DESC
+			  LIMIT ?
+			)
+		AND user_id = ?
+		AND seen_at != 0;`
+
+		result, err := dbSession.Exec(sql, userId, maxConcurrentSessions-1, userId)
+		if err != nil {
+			return err
+		}
+
+		rowsAffected, err := result.RowsAffected()
+		if err == nil && rowsAffected > 0 {
```

Comment:
Is it safe to ignore errors here?

Reply:
It basically returns a `errors.New("AffectedArrows is not supported by this driver")` error or a similar one for DDL statements (create, alter, drop database, etc). So... yes? 😇 

If an error happens, the `Exec` method should return it, I guess.

Reply:
:+1: Thanks!